epel/map.jinja

```diff
@@ -38,6 +38,12 @@
         'key_name': 'RPM-GPG-KEY-EPEL-7',
         'rpm': 'http://download.fedoraproject.org/pub/epel/7/x86_64/Packages/e/epel-release-7-11.noarch.rpm',
       },
+      '8': {
+        'key': 'https://fedoraproject.org/static/352C64E5.txt',
+        'key_hash': 'sha256=22f25ad95d5e8d371760815485dba696ea3002fc2c7f812f2c75276853387107',
```

Comment:
The `key` and `key_hash` is the same as `EPEL (7)`.  Updated ones aren't available for `EPEL (8)` so far?

Reply:
Ow wait, that's not right! Must be a copy paste / selection buffer error.

thanks for spotting that.